src/ert/gui/tools/plot/plottery/plots/histogram.py

```diff
@@ -33,14 +33,13 @@ def plot(
         observation_data: pd.DataFrame,
         std_dev_images: dict[str, npt.NDArray[np.float32]],
     ) -> None:
-        plotHistogram(figure, plot_context, ensemble_to_data_map, observation_data)
```

Comment:
Probably want the commit title `Remove unused parameter observations_data` here, as the verb `delete` is a bit unconventional.